pkg/lib/host-connector/types/pcr_manifest.go

```diff
@@ -38,21 +39,56 @@ type EventLog struct {
 	Info       map[string]string `json:"info"`
 }
 
+type EventLogCriteria struct {
+	TypeID      string   `json:"type_id"`   //oneof-required
+	TypeName    string   `json:"type_name"` //oneof-required
+	Tags        []string `json:"tags,omitempty"`
+	Measurement string   `json:"measurement"` //required
+}
+
 type EventLogEntry struct {
 	PcrIndex  PcrIndex     `json:"pcr_index"`
 	EventLogs []EventLog   `json:"event_log"`
 	PcrBank   SHAAlgorithm `json:"pcr_bank"`
 }
 
+type EventLogEntryFC struct {
+	Pcr      PCR                `json:"pcr"`
+	TpmEvent []EventLogCriteria `json:"tpm_events"`
+}
+
+type PCR struct {
+	Index int    `json:"index"`
+	Bank  string `json:"bank"`
+}
+
+type PCRS struct {
+	PCR              PCR                `json:"pcr"`         //required
+	Measurement      string             `json:"measurement"` //required
+	PCRMatches       bool               `json:"pcr_matches,omitempty"`
+	EventlogEqual    *EventLogEqual     `json:"eventlog_equals,omitempty"`
+	EventlogIncludes []EventLogCriteria `json:"eventlog_includes,omitempty"`
+}
+
+type EventLogEqual struct {
+	Events      []EventLogCriteria `json:"events,omitempty"`
+	ExcludeTags []string           `json:"exclude_tags,omitempty"`
+}
+
 type PcrEventLogMap struct {
 	Sha1EventLogs   []EventLogEntry `json:"SHA1"`
 	Sha256EventLogs []EventLogEntry `json:"SHA256"`
 }
 
+type PcrEventLogMapFC struct {
```

Comment:
remove duplicate structures

Reply:
New structure to maintain both exsi and linux flow.

Reply:
Why old has not been removed then?

Reply:
PcrEventLogMap struct is for Esxi flow and PcrEventLogMapFC is for Linux flow(Based on new design).